### PR TITLE
Feat/distributed otel

### PR DESCRIPTION
**Pull request by ankitnayan**
distributed ingestion to clickhouse in place

### CODE REVIEW

File: config/default-config.yaml [hunk 1]
```diff
@@ -1,76 +1,37 @@
 receivers:
-  otlp/spanmetrics:
```
Comment (makeavish) [inline config/default-config.yaml:2]: @ankitnayan This file should not be edited

File: exporter/clickhouselogsexporter/migrations/000001_init_db.down.sql [hunk 2]
```diff
@@ -1,7 +1,7 @@
-DROP TABLE IF EXISTS resource_keys_string_final_mv;
-DROP TABLE IF EXISTS atrribute_keys_float64_final_mv;
-DROP TABLE IF EXISTS atrribute_keys_int64_final_mv;
-DROP TABLE IF EXISTS atrribute_keys_string_final_mv;
```
Comment (makeavish) [inline exporter/clickhouselogsexporter/migrations/000001_init_db.down.sql:4]: Why are we not dropping MVs?

**Comment by srikanthccv** [comment]
I get this error when I restart the collector locally. Is it known/fixed?

```
Error: cannot build pipelines: failed to create "clickhousetraces" exporter, in pipeline "traces": Clickhouse Migrate failed to run, error: code: 57, message: There was an error on [clickhouse-2:9000]: Code: 57. DB::Exception: Table default.schema_migrations already exists. (TABLE_ALREADY_EXISTS) (version 22.8.8.3 (official build)) in line 0:
			CREATE TABLE schema_migrations ON CLUSTER cluster (
				version    Int64,
				dirty      UInt8,
				sequence   UInt64
			) Engine=MergeTree ORDER BY sequence
```

**Comment by makeavish** [comment]
@srikanthccv How are you running it? On k8s or docker? Are you specifying password/username?

**Comment by srikanthccv** [comment]
I am using docker, and I didn't specify any username/passwd.

**Comment by makeavish** [comment]
Looks like a problem with migrate command as in log shared by you database is not being specified.
Thanks for pointing it out, I will test on local once to fix it.

File: exporter/clickhouselogsexporter/migrations/000001_init_db.down.sql (hunk 2 above)
Comment (ankitnayan) [reply exporter/clickhouselogsexporter/migrations/000001_init_db.down.sql:4]: done

**Pull request merged**